calyx-backend/src/firrtl.rs

```diff
@@ -84,41 +92,106 @@ fn emit_component<F: io::Write>(
     // Add a COMPONENT START: <name> anchor before any code in the component
     writeln!(f, "{}; COMPONENT START: {}", SPACING.repeat(2), comp.name)?;
 
-    // TODO: Cells. NOTE: leaving this one for last
+    // Cells
+    for cell in comp.cells.iter() {
+        let cell_borrowed = cell.as_ref().borrow();
+        if cell_borrowed.type_name().is_some() {
+            match cell_borrowed.prototype {
+                ir::CellType::Primitive {
+                    name: _,
+                    param_binding: _,
+                    is_comb: _,
+                    latency: _,
+                } => {
+                    // TODO: use extmodules
+                    writeln!(
+                    f,
+                    "{}; FIXME: attempting to instantiate primitive cell {}",
+                    SPACING.repeat(2),
+                    cell_borrowed.name()
+                )?
```

Comment:
This is super minor, but it kinda looks like some indentation was lost here?

Reply:
Yup, the indentation was a bit wonky here :sweat_smile: Thanks for pointing it out!!!